NEWS.md

```diff
@@ -1,3 +1,7 @@
+# 2.5.1 2022-08-26
```

Comment:
Was the release made on the 26th?

Reply:
Yes, this pull request for the release has been created on 2022-08-26.

Reply:
That wasn't really my question. I asked if the release had been made on that date.

As far as I can tell the release did not happen (not notice sent on slack or tag in this repo)